Jenkinsfile

```diff
@@ -0,0 +1,3 @@
+// Builds a module using https://github.com/jenkins-infra/pipeline-library
+// TODO: Explicitly require build on the latest Weekly in addition to the the target LTS
```

Comment:
Do not have it build the latest weekly. You can have it build a _specific_ weekly.